### PR TITLE
Dropdown outside click

### PR DESCRIPTION
**Pull request by audreyli**
Kapture 2021-01-24 at 20 38 32

### CODE REVIEW

File: src/components/Toolbar/ActionMenu.js [hunk 1]
```diff
@@ -24,15 +25,31 @@ import React, {Component} from 'react';
...
+
   constructor() {
     super();
     this.state = {
       active: false,
     };
   }
 
+  handleMouseDown = (e) => {
+    const refNode = findDOMNode(this.containerRef.current);
```
Comment (stevenhao) [inline src/components/Toolbar/ActionMenu.js:38]: What happens if you don't use `findDOMNode` here?

Comment (audreyli) [reply src/components/Toolbar/ActionMenu.js:38]: it'll set it to close and then handleClick on the dropdown menu which triggers after will open it (so clicking on the button no longer closes the menu). couldn't think of a nice way to handle that case other than checking the node

Comment (stevenhao) [reply src/components/Toolbar/ActionMenu.js:38]: wow that's rough. does `mouseMove={e=>e.stopPropagation()}` do anything?

Comment (audreyli) [reply src/components/Toolbar/ActionMenu.js:38]: to handle this case? it shouldnt, the mouse doesnt actually move between mousedown and mouseup. also prob not good to randomly stop propagation of global mouse move event :P 

im also happy to close this in favor of using a library (material-ui) that already supports outside click to close

Comment (stevenhao) [reply src/components/Toolbar/ActionMenu.js:38]: i meant attaching that to the dropdown menu itself, so that the event doesn't bubble up to window

**Pull request merged**